specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -69,53 +69,53 @@ paths:
           schema:
             type: "string"
         - in: query
-          name: appartementsrechtvolgnummer
+          name: kadastraleaanduiding__appartementsrechtvolgnummer
           description: "Nummer dat het kadastraal object uniek identificeert als een\
             \ appartementsrecht binnen het complex."
           schema:
             type: "integer"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
```

Comment:
ingeschrevenpersoon is geen property van de kadastraal onroerende zaak. Het complete pad is zakelijkgerechtigden__persoon__identificatie. Maar of het daar nou duidelijker van wordt? (niet dus, naam mijn mening)

Mijn voorstel zou zijn zakelijkgerechtigde__burgerservicenummer

Reply:
De zakelijkgerechtigde heeft geen burgerservicenummer als element dus dat is ook niet echt eenduidig en wordt door onze naamgevingsconventie niet afgedekt.  Best wel een dingetje. Dat had ik gisteren ook ontdekt en heb in #254 daar ook een comment aan gewijd met mijn overwegingen.